presto-tests/src/test/java/io/prestosql/execution/TestQueryRunnerUtil.java

```diff
@@ -64,7 +64,7 @@ public static void waitForQueryState(DistributedQueryRunner queryRunner, QueryId
                     dispatchManager.getQueryInfo(queryInfo.getQueryId());
                 }
             }
-            MILLISECONDS.sleep(500);
+            MILLISECONDS.sleep(100);
```

Comment:
related?

Reply:
It's on a separate commit.  Not necessary, just trying to speed up tests - waiting half a second to check query state changes seemed excessive.  Do you think 100ms is too short?

Reply:
> It's on a separate commit.

nvrm, i missed that